packages/components/src/Select/Select.utils.js

```diff
@@ -0,0 +1,35 @@
+import { is } from '@wp-g2/utils';
+import React from 'react';
+
+const isGroup = (option) => {
+	return is.object(option) && is.array(option.options);
+};
+
+const renderOption = (option = {}, index) => {
+	const { disabled, id, label, value } = option;
+	const key = id || value || index;
+
+	return (
+		<option disabled={disabled} key={key} value={value}>
+			{label}
+		</option>
+	);
+};
+
+export const renderContent = ({ children, options }) => {
+	if (children) return children;
```

Comment:
I don't fully understand how/where `children` would come from here.

Reply:
`children` would come from something like this:

```jsx
<Select>
  <option>...</option>
  <option>...</option>
  <option>...</option>
</Select>
```

☝️ a manually written collection of `options`, rather than using the `options` prop.

Reply:
> a manually written collection of option

Gotcha! I didn't think that the component would work that way but it makes sense.